addons/website_customer/views/website_customer_view.xml

```diff
@@ -1,20 +1,6 @@
 <?xml version="1.0"?>
 <openerp>
     <data>
-        <record id="view_partners_form_website" model="ir.ui.view">
-            <field name="name">view.res.partner.form.website.tags</field>
-            <field name="model">res.partner</field>
-            <field name="inherit_id" ref="website_partner.view_partners_form_website" />
-            <field name="arch" type="xml">
-                <data>
-                    <xpath expr="//group[@name='mailing']" position="after">
-                        <group name="website" string="Website">
-                            <field name="tag_ids" widget="many2many_tags"/>
```

Comment:
@tde-banana-odoo We are created already new task for add tags from front-end.
Task : https://www.odoo.com/web#id=16142&view_type=form&model=project.task&action=333&active_id=131